docs/guides/nrfconnect_examples_configuration.md

```diff
@@ -0,0 +1,136 @@
+# Configuring nRF Connect SDK examples
+
+When you tried to build nRF Connect SDK example applications like
+[lock-app](../../examples/lock-app/nrfconnect/README.md),
+[lighting-app](../../examples/lighting-app/nrfconnect/README.md) or
+[pigweed-app](../../examples/pigweed-app/nrfconnect/README.md), you could notice
+that some default configuration is used then. Probably after testing an example
+solution you may want to change something in the application configuration or
+add new functionalities to build your own application based on sample. This page
+contains information about how to configure an application in different ways and
+also sheds some light on the configuration structure to allow you understand
+building process better.
+
+<hr>
+
+## Configuring application
+
+Changing default application configuration might be done in the many ways and
+you should choose the one that suits you, depending on your needs. You are
+allowed to change configuration temporarily, what will be useful for testing
+impact of changes on the application behavior, but if you want to develop own
+application, probably you will be interested in making permanent changes.
```

Comment:
@greg-fer is that okay to use sentences like: "probably you will be interested"? I would use different form of sentence to inform about the possibility to make permanent changes.

Reply:
Done